CHANGELOG/CHANGELOG-1.26.md

```diff
@@ -1850,4 +1850,4 @@ name | architectures
 - go.opentelemetry.io/otel/exporters/otlp: v0.20.0
 - go.opentelemetry.io/otel/oteltest: v0.20.0
 - go.opentelemetry.io/otel/sdk/export/metric: v0.20.0
-- go.opentelemetry.io/otel/sdk/metric: v0.20.0
\ No newline at end of file
+- go.opentelemetry.io/otel/sdk/metric: v0.20.0
```

Comment:
I think this change is extra, right? Because anything changed in this line.
Could you remove it if this is the case?